applications/openshift/api-server/api_server_tls_security_profile/rule.yml

```diff
@@ -0,0 +1,48 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: "Check if API server is configured to use Old tlsSecurityProfile"
+
+description: |-
+    The configuration <tt>tlsSecurityProfile</tt> specifies TLS configurations
+    to be used while establishing connections with the externally exposed
+    servers. Though secure transport mode is used for establishing connections,
+    the protocols used may not always be strong enough to avoid interception and
+    manipulation of the data in transport. TLS Security profile configured should 
+    not make use of any protocols, ciphers, and algorithms with known security
+    vulnerabilities.
+
+    Update tlsSecurityProfile to Intermediate using the following command:
```

Comment:
STIG SRG Mapping sheet had the fix is to set the tlsSecurityProfile to Intermediate, in case Old was used or Custom profile defined had TLS version less than 1.2.
If the rule suggests to rely on default, should the suggested command be for removing the configured tlsSecurityProfile?

Reply:
Good point. Let's discuss this in the sheet.

Reply:
I asked in the sheet, but David is out of the office until the end of the month. Let's add a sub-task in Jira to track this and in the meantime proceed with what you have since it aligns with the sheet.